packages/app/app/actions/favorites.ts

```diff
@@ -61,6 +62,19 @@ export function removeFavoriteTrack(track) {
   };
 }
 
+export function reorderFavoriteTrack(fromIndex, toIndex) {
```

Comment:
To be honest, when I read the feature request, I thought sorting would only change the order the tracks are displayed in, and I didn't think permanently saving them in this state would be necessary. Do you think it would be better to do it this way?